app/script/nodes/Image/Effects/NodeHueShift.ts

```diff
@@ -0,0 +1,79 @@
+///<reference path="../NodeImageTransformation.ts" />
+
+/**
+ * @classdesc "Hue" shift the colours on an image by a given amount
+ * @author Simon, Orlando
+ */
+
+@register
+class NodeHueShift extends NodeImageTransformation {
```

Comment:
Will need to export now, add the relevant imports, and remove the `///reference`